packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { CommunicationIdentifierKind } from '@azure/communication-common';
+import { Icon, mergeStyles, Stack, useTheme } from '@fluentui/react';
+import React, { useRef } from 'react';
+import { FileUploadHandler } from './FileUploadHandler';
+import { UploadedFile } from './UploadedFile';
+
+/**
+ * Props for {@link FileUploadButton} component.
+ * @internal
+ */
+export interface FileUploadButtonProps {
+  /**
+   * The ACS user ID of the user uploading the file.
+   */
+  userId: CommunicationIdentifierKind;
+  /**
+   * The function of type {@link FileUploadHandler} for handling file uploads.
+   */
+  fileUploadHandler?: FileUploadHandler;
+  /**
+   * A string containing the comma separated list of accepted file types.
+   * Similar to the `accept` attribute of the `<input type="file" />` element.
+   */
+  accept?: string;
+  /**
+   * Allows multiple files to be selected if set to `true`.
+   * Default value is `false`.
```

Comment:
Worth documenting here has well that this is similar to `multiple` attribute of `<input type="file" />`